server/app/services/question/QuestionService.java

```diff
@@ -393,13 +393,6 @@ private ImmutableSet<CiviFormError> validateQuestionImmutableMembers(
                   questionDefinition.getQuestionPathSegment(), toUpdate.getQuestionPathSegment())));
     }
 
-    if (!questionDefinition.getQuestionType().equals(toUpdate.getQuestionType())) {
```

Comment:
```
java.lang.RuntimeException: com.fasterxml.jackson.databind.exc.InvalidTypeIdException: Could not resolve type id 'name' as a subtype of `services.question.types.AutoValue_AddressQuestionDefinition_AddressValidationPredicates`: Class `services.question.types.AutoValue_NameQuestionDefinition_NameValidationPredicates` not subtype of `services.question.types.AutoValue_AddressQuestionDefinition_AddressValidationPredicates`
```

Reply:
what happens if someone POSTs a question definition with the type changed?

Reply:
Is it possible for an admin to do that? The only way to edit a question that I know of is from within the question, at which point you're already using the same question type. But if it is possible, it will result in the Jackson type error I copied above, so I could add some logic to handle that.

Reply:
Oh I'm realizing now maybe you meant if they're importing a question with the same admin name, but a different type? I'll handle that when I do the work to handle admin name collisions, but the guidance from product is to make a new question of the new type and leave it up to admin to swap back the old question. We will want to give them some warning that that's happening though.